src/pages/register-team.tsx

```diff
@@ -4,12 +4,12 @@ import React from "react";
 import ProtectedPage from "../components/ProtectedPage";
 import TeamRegistrationForm from "../components/TeamRegistrationForm";
 
-import * as styles from "../components/form/Form.module.scss";
-
 export default function TeamRegistration({ data }) {
-  const handles = new Set(data.handles.edges.map((h) => h.node.handle));
+  const handles: Set<string> = new Set(
+    data.handles.edges.map((h) => h.node.handle)
+  );
 
-  let wardens: { value: string; image: string }[] = [];
+  let wardens: { value: string; image: unknown }[] = [];
```

Comment:
Similar question -- probably has other effects

```suggestion
  let wardens: { value: string; image: string | undefined }[] = [];
```

Reply:
Yeah, same here. We could define the Sharp Image object, but I thought it was fine as an intermediate step (where we have some typescript sprinkled in with javascript) to just say the type is unknown. What do you think?

Reply:
Later's fine -- I misunderstood what these were.